src/diagnostics/edmfx_diagnostics.jl

```diff
@@ -1272,10 +1272,18 @@ function compute_edt!(
     vert_diff::Union{VerticalDiffusion, DecayWithHeightDiffusion},
     turbconv_model::Nothing,
 )
+    (; vert_diff) = cache.atmos
+    (; ᶜp) = cache.precomputed
+
+    if vert_diff isa DecayWithHeightDiffusion
+        ᶜK_h = compute_eddy_diffusivity_coefficient(state.c.ρ, vert_diff)
```

Comment:
Are you missing `@.` here?  Something like `ᶜK_h = @. compute_eddy_diffusivity_coefficient(state.c.ρ, vert_diff)` Is the diagnostic output correct?

Reply:
`compute_eddy_diffusivity_coefficient` returns a @. lazy eddy_diffusivity_coefficient(....) so it should be broadcasted

Reply:
Yeah. As i mentioned below. We need a naming convention to be able to quickly tell if its a `lazy` function or a field, or a function. Otherwise it is difficult to review this stuff efficiently